pulp_rpm/app/models/repository.py

```diff
@@ -51,6 +51,12 @@ class RpmRemote(Remote):
     TYPE = "rpm"
     sles_auth_token = models.CharField(max_length=512, null=True)
 
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
+
+    uln_server_base_url = models.CharField(max_length=512, null=True)
```

Comment:
@quba42 , @dkliban , @ThikaXer , @dralley , @bmbouter , was it discussed in any way if it's worth having a separate remote type for ULN support?
It adds a lot to a remote and all users will see this. I'm not sure, just asking if it was discussed.

Reply:
I have actually also opened a thread to discuss this further up. I would say it is certainly an option. However, to form a clear opinion, I need to have some questions about the fields in the Remote base class answered:

1. There is a `username` and `password` field in the remote base class. Could those be used _instead_ of adding `uln_user` and `uln_pwd`? (This would reduce the amount of stuff added to the RpmRemote.)
2. What is the logic behind adding things like `username` and `password` to the remote base class, given that "It adds a lot to a remote and all users will see this"?

I do feel like a separate ULN remote would have the advantage that fields required for ULN repositories could actually be required fields, which would reduce the need for explicit exceptions handling.

Reply:
On the other hand the `sles_auth_token` is also part of the `RpmRemote`, so why shouldn't `uln_server_base_url` be?
(I am still unclear whether `uln_user` and `uln_pwd` are needed.)

Reply:
Having discussed this matter with @quba42, I would be interested in your opinions on the questions above. Maybe more experienced pulp developers can answer these? I would appreciate any comments or opinions on how to handle this. Thank you :)

Reply:
Done.